spec/factories/users.rb

```diff
@@ -9,5 +9,7 @@
     sequence :username do |n|
       Faker::Internet.username.to_s + n.to_s
     end
+    bio { Faker::Lorem.paragraph(sentence_count: 0) }
+    website { Faker::Lorem.paragraph(sentence_count: 0) }
```

Comment:
Aren't this generating empty strings? I would add valid values that are not blank.